docs/content/docs/operators/stats/chisqtest.md

```diff
@@ -0,0 +1,187 @@
+---
+title: "ChiSqTest"
+type: docs
+aliases:
+- /operators/stats/chisqtest.html
+---
+
+<!--
+Licensed to the Apache Software Foundation (ASF) under one
+or more contributor license agreements.  See the NOTICE file
+distributed with this work for additional information
+regarding copyright ownership.  The ASF licenses this file
+to you under the Apache License, Version 2.0 (the
+"License"); you may not use this file except in compliance
+with the License.  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing,
+software distributed under the License is distributed on an
+"AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+KIND, either express or implied.  See the License for the
+specific language governing permissions and limitations
+under the License.
+-->
+
+## ChiSqTest
+
+Chi-square Test computes the statistics of independence of variables in a contingency table,
+e.g., p-value, and DOF(degree of freedom) for each input feature. The contingency table is
+constructed from the observed categorical values.
+
+### Input Columns
+
+| Param name  | Type   | Default      | Description                |
+|:------------|:-------|:-------------|:---------------------------|
+| featuresCol | Vector | `"features"` | The feature vector.        |
```

Comment:
nit: could you remove `The` in the description to be consistent with existing docs?

Same for other params.